sitemedia/scss/components/_searchform.scss

```diff
@@ -47,20 +51,34 @@
             margin-top: spacing.$spacing-sm;
         }
         label[for="sort"] {
-            @include container.two-column;
+            max-width: 100%;
             order: 3;
             flex: 1 0 100%;
             margin-top: spacing.$spacing-lg;
         }
-        input[type="submit"] {
-            @include typography.button;
+        button[type="submit"] {
             order: 2;
-            flex: 0 0 10%;
+            flex: 0 0 40px;
             border-radius: 0;
             margin-left: 0;
             background-color: colors.$gray-30;
             border: 1px solid colors.$gray-50;
-            height: spacing.$spacing-xl;
+            font-size: typography.$text-size-xl;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            @include a11y.sr-only;
+            @include a11y.sr-only-focusable;
```

Comment:
Do we need both? Should `sr-only-focusable` imply `sr-only` ?

I see from the note on the existing class/mixin that they're meant to be applied topegher, just wondering if you can think of any case when we would want `sr-only-focusable` without `sr-only`

Reply:
I can't, so maybe good to go ahead and just include the `sr-only` mixin inside of the `focusable` one.